circle.yml

```diff
@@ -0,0 +1,3 @@
+machine:
+  node:
+    version: 7
```

Comment:
Should be 6.10.2, or at least ~6, to match production: https://github.com/resin-io/resin-base/blob/master/Dockerfile#L41